### PR TITLE
Update existing packages 

### PR DESCRIPTION
**Pull request by allenwq**
Webpack is upgraded to 2.0 and the configuration file is changed.

### CODE REVIEW

File: client/.eslintrc [hunk 1]
```diff
@@ -2,10 +2,10 @@
   "parser": "babel-eslint",
   "extends": "airbnb",
   "settings": {
-    import/resolver: "webpack"
+    "import/resolver": "webpack"
```
Comment (allenwq) [inline client/.eslintrc]: @kxmbrian The configuration here does not seem to affect anything, do you know what's this was used for?

Comment (kxmbrian) [inline client/.eslintrc]: In both pre and post update, eslint cannot resolve `lib` without the config. I get errors like: 
```
  1:1   error  'lib' should be listed in the project's dependencies. Run 'npm i -S lib' to add it  import/no-extraneous-dependencies
  1:33  error  Unable to resolve path to module 'lib/helpers/table_row_form_helpers'               import/no-unresolved
  1:33  error  Missing file extension for "lib/helpers/table_row_form_helpers"                     import/extensions
```

You don't get them when you remove the setting and run `npm run lint`?

Comment (allenwq) [inline client/.eslintrc]: have you upgraded eslint ?  it's OK for me and you see the test suite passed.

Comment (kxmbrian) [inline client/.eslintrc]: You tried removing all of `settings: {...}` right? Not sure if I missed something, but I haven't managed to get it to work on my local and it's failing on #2017.

Comment (allenwq) [inline client/.eslintrc]: Oh I didn't remove all of the settings ><  Sorry this is my bad, should be more clear.

Comment (kxmbrian) [inline client/.eslintrc]: ok, so let's keep this config?

Comment (allenwq) [inline client/.eslintrc]: Yes !

**Comment by kxmbrian** [comment]
LGTM!

**Pull request merged**